If check is detected, it makes the builds non-determinstic so just force
it to be disabled.

RP - 4/11/08

Index: parted-1.8.8/configure.ac
===================================================================
--- parted-1.8.8.orig/configure.ac	2008-11-04 15:18:17.000000000 +0000
+++ parted-1.8.8/configure.ac	2008-11-04 15:18:31.000000000 +0000
@@ -481,6 +481,6 @@
 AM_CONDITIONAL([COMPILE_FOR_S390], [test "$compile_for_s390" = "yes"])
 
 dnl check for "check", unit testing library/header
-PKG_CHECK_MODULES([CHECK], [check >= 0.9.3], have_check=yes, have_check=no)
+have_check=no
 if test "$have_scintilla" != "yes"; then
     AC_MSG_RESULT([Unable to locate check version 0.9.3 or higher: not building])
